beater/otlp/grpc.go

```diff
@@ -35,35 +34,24 @@ import (
 )
 
 var (
-	monitoringKeys = append(request.DefaultResultIDs,
-		request.IDResponseErrorsRateLimit,
-		request.IDResponseErrorsTimeout,
-		request.IDResponseErrorsUnauthorized,
-	)
 	gRPCMetricsRegistry      = monitoring.Default.NewRegistry("apm-server.otlp.grpc.metrics")
 	gRPCMetricsMonitoringMap = request.MonitoringMapForRegistry(gRPCMetricsRegistry, monitoringKeys)
 	gRPCTracesRegistry       = monitoring.Default.NewRegistry("apm-server.otlp.grpc.traces")
 	gRPCTracesMonitoringMap  = request.MonitoringMapForRegistry(gRPCTracesRegistry, monitoringKeys)
 	gRPCLogsRegistry         = monitoring.Default.NewRegistry("apm-server.otlp.grpc.logs")
 	gRPCLogsMonitoringMap    = request.MonitoringMapForRegistry(gRPCLogsRegistry, monitoringKeys)
 
-	// RegistryMonitoringMaps provides mappings from the fully qualified gRPC
+	// GRPCRegistryMonitoringMaps provides mappings from the fully qualified gRPC
 	// method name to its respective monitoring map.
-	RegistryMonitoringMaps = map[string]map[request.ResultID]*monitoring.Int{
+	GRPCRegistryMonitoringMaps = map[string]map[request.ResultID]*monitoring.Int{
 		metricsFullMethod: gRPCMetricsMonitoringMap,
 		tracesFullMethod:  gRPCTracesMonitoringMap,
 		logsFullMethod:    gRPCLogsMonitoringMap,
 	}
 )
 
-const (
-	metricsFullMethod = "/opentelemetry.proto.collector.metrics.v1.MetricsService/Export"
-	tracesFullMethod  = "/opentelemetry.proto.collector.trace.v1.TraceService/Export"
-	logsFullMethod    = "/opentelemetry.proto.collector.logs.v1.LogsService/Export"
-)
-
 func init() {
-	monitoring.NewFunc(gRPCMetricsRegistry, "consumer", collectMetricsMonitoring, monitoring.Report)
+	monitoring.NewFunc(gRPCMetricsRegistry, "gRPCConsumer", collectMetricsMonitoring, monitoring.Report)
```

Comment:
```suggestion
	monitoring.NewFunc(gRPCMetricsRegistry, "consumer", collectMetricsMonitoring, monitoring.Report)
```

grpc is already in the registry name

Reply:
Solved in 7c3efe2e6